packages/blade/src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import type { TabsProps } from './types';
+import { TabsContext } from './TabsContext';
+import { useControllableState } from '~utils/useControllable';
+import { useId } from '~utils/useId';
+import BaseBox from '~components/Box/BaseBox';
+
+const Tabs = ({
```

Comment:
Add basic usage example in JSDoc here maybe?

Reply:
Added